resources/validations/src/ssp.sch

```diff
@@ -295,26 +334,26 @@
             <sch:let name="corrections"
                      value="lv:correct($registry/f:fedramp-values/f:value-set[@name = 'control-implementation-status'], $status)" />
             <sch:let name="required-response-points"
-                     value="$selected-profile/o:catalog//o:part[@name = 'item']" />
-            <sch:let name="implemented"
-                     value="/o:system-security-plan/o:control-implementation/o:implemented-requirement/o:statement" />
-            <sch:let name="missing"
-                     value="$required-response-points[not(@id = $implemented/@statement-id)]" />
+                     value="$selected-profile/o:catalog//o:control[@id = current()/@control-id]//o:part[@name = 'item'][o:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'response-point']]" />
             <sch:assert diagnostics="invalid-implementation-status-diagnostic"
                         doc:organizational-id="section-c.2"
                         id="invalid-implementation-status"
                         role="error"
                         test="not(exists($corrections))">[Section C Check 2] Implementation status is correct.</sch:assert>
-            <sch:report id="implemented-response-points"
-                        role="information"
-                        test="exists($implemented)">[Section C Check 2] This SSP has implemented a statement for each of the following lettered
-                        response points for required controls: 
-            <sch:value-of select="$implemented/@statement-id" />.</sch:report>
+            <sch:report role="information"
```

Comment:
The UI is grouping the asserts and reports by ID - could we maintain an ID for each, or are there considerations that would prevent that?

Reply:
I added an id to that (single) `sch:report`. There were no others.

I have prepared but not yet committed Schematron for FedRAMP Schematron documents and XSpec documents. It will enforce rules such as "assertions must have an id".